tests/basictests/ml-pipelines.sh

```diff
@@ -47,6 +47,17 @@ function check_run_status() {
     os::cmd::try_until_text "curl ${SVC}/apis/v1beta1/runs/${RUN_ID} | jq '.run.status'" "Completed" $odhdefaulttimeout $odhdefaultinterval
 }
 
+function setup_monitoring() {
+    header "Enabling User Workload Monitoring on the cluster"
+    oc apply -f ${RESOURCEDIR}/modelmesh/enable-uwm.yaml
+}
+
+function test_metrics() {
+    header "Checking metrics for total number of runs, should be 1 since we have spun up 1 run"
+    monitoring_token=`oc sa get-token prometheus-k8s -n openshift-monitoring`
```

Comment:
I'm pretty sure this command will be breaking soon enough (openshift 4.11) @crobby we should consider an alternative command here right?

Reply:
Where can I find documentation for OCP 4.11? Trying to look up for an alternative 4.11 supporting command but unable to find relevant info.

Also would the alternative 4.11 supporting command work for the earlier versions of OCP? Or will we just shift to 4.11 and not use earlier versions at all?

Reply:
Do I need to replace ``` with `$()` around the command to make it OCP 4.11 compatible, like @crobby has done in his PR here? https://github.com/opendatahub-io/odh-manifests/blob/0bb13e5aa65f63e9b53b35a025ea3d2f746d0057/tests/basictests/modelmesh.sh#L65

Reply:
Nah, you will probaably beed to manually grab the token from the secret associated with the service account. 

I am "fine" with us leaving it as is for now -- I think if tests break once we switch out ci over to 4.11 this is something we'd investigate.

Reply:
Yeah, this should be fine.  I think it's deprecated functionality and we will replace all instances of this pattern with `oc create-token ...` in the future.